eslint.config.js

```diff
@@ -14,5 +14,5 @@ export default [
     },
   },
   { rules: { "no-console": "error" } },
-  { ignores: ["dist/**", ".astro", "public/pagefind/**"] },
+  { ignores: ["dist/**", "**/*.astro", "public/pagefind/**"] },
```

Comment:
We're ignoring `.astro` directory which is auto-generated when we run the project. However, we need to lint all other astro files.

To test this, create an unused variable in one of astro files. Then, run the `pnpm lint` and you'll see the issue.

```js
// file: src/layouts/PostDetails.astro

const test = ""; // this var is not being used and eslint will warn us

```

However, if we ignore all astro files, `"**/*.astro"`, we won't get the warnings anymore and this is not what we want.

Reply:
So, I'm closing this PR for now.

Reply:
I understand, but there's an issue when you use code from Google Tag Manager/Analytics. So, I thought you've met this issue too and tried to disable linting for astro files :)